src/Psy/Shell.php

```diff
@@ -1013,7 +1013,7 @@ protected function writeVersionInfo()
         try {
             $client = $this->config->getChecker();
             if (!$client->isLatest()) {
-                $this->output->writeln(sprintf('New version is available (current: %s, latest: %s)',self::VERSION, $client->getLatest()));
+                $this->output->writeln(sprintf('New version is available (current: %s, latest: %s)', self::VERSION, $client->getLatest()));
```

Comment:
This was not fixed by StyleCI because the fixer is turned off in the configuration.

Reply:
still, PSR2 violation

Reply:
this is `method_argument_space`, right? is that the one we disabled because it prevents lining up arguments on consecutive lines?

Reply:
yes. yet, i didn't enabled the rule or apply it globally, this was the line where space was simply missing, and you are talking about cases where there were multiple spaces for aligning purposes